site/e2e/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import { Page } from "@playwright/test"
+
+export const buttons = {
+  starterTemplates: "Starter templates",
+  dockerTemplate: "Develop in Docker",
+  useTemplate: "Use template",
+  createTemplate: "Create template",
+  createWorkspace: "Create workspace",
+  submitCreateWorkspace: "Create workspace",
+  stopWorkspace: "Stop",
+  startWorkspace: "Start",
+}
```

Comment:
I'm not sure I would decouple this page-specific constant from the test for that page.